pkg/sql/delegate/show_table.go

```diff
@@ -135,7 +138,7 @@ WHERE
     AND table_schema=%[5]s
     AND table_name=%[2]s
 ORDER BY
-    1, 2, 3, 4, 5, 6, 7, 8;`
+    1, 2, 4;`
```

Comment:
was it on purpose you removed the other ordering columns?

Looks to me like this will make the output non-deterministic.

Reply:
I asked asked Wenyi to make this change ([here](https://github.com/cockroachdb/cockroach/pull/83388#issuecomment-1185439749)). Columns 1, 2, 4 are the natural key of the query, and will be unique.